src/components/AvatarWithIndicator.js

```diff
@@ -22,9 +22,9 @@ import themeColors from '../styles/themes/default';
 const propTypes = {
     /** URL for the avatar */
     source: PropTypes.oneOfType([PropTypes.string, PropTypes.func]).isRequired,
-
-    /** To show a tooltip on hover */
-    tooltipText: PropTypes.string,
+    
+     /** To show a tooltip on hover */
+     tooltipText: PropTypes.string,
```

Comment:
I believe this should be replaced by accountID

Reply:
This component seems to be poorly named. I thought the same at first, but this avatar is only used on the sidebar header, where it links to settings. So I think a tooltipText is more appropriate here (or it really will not be clear it takes you to settings)
<img width="250" alt="Screenshot 2023-06-10 at 4 32 44 PM" src="https://github.com/Expensify/App/assets/59715908/d2009750-e73b-40a0-8272-f6905ad2948a">

Reply:
This is only use of this component 

https://github.com/Expensify/App/blob/035783a7984e0214677b5364d9351c968887bf58/src/pages/home/sidebar/SidebarLinks.js#LL202C33-L205C35